### PR TITLE
SB3 2166: Inputs, buttons, patching

### PR DESCRIPTION
**Pull request by rhiannanberry**
Covers 2 tickets (plus a little more)
https://springbot.atlassian.net/browse/SB3-2167
https://springbot.atlassian.net/browse/SB3-2166

- Unifying styles between styles and traits
   - specifically: select, input, buttons, checkboxes
- hosting Roboto locally to try and fix some weird font rendering issues
- fix for [this bug](https://github.com/artf/grapesjs/issues/3735#issuecomment-913921482) introduced in the current tag.
- Removing the new temp property type I added before (UnitValue property)

### CODE REVIEW

File: src/trait_manager/index.js [hunk 1]
```diff
@@ -100,6 +100,7 @@ export default () => {
         config: c
       });
       TraitsViewer.itemsView = types;
+      TraitsViewer.updatedCollection();
```
Comment (rhiannanberry) [inline src/trait_manager/index.js:103]: Relates to [this issue](https://github.com/artf/grapesjs/issues/3735#issuecomment-913921482)

File: src/trait_manager/view/TraitsView.js [hunk 2]
```diff
@@ -15,7 +15,6 @@ export default DomainViews.extend({
     this.className = this.pfx + 'traits';
     const toListen = 'component:toggled';
     this.listenTo(this.em, toListen, this.updatedCollection);
-    this.updatedCollection();
```
Comment (rhiannanberry) [inline src/trait_manager/view/TraitsView.js:18]: Relates to [this issue](https://github.com/artf/grapesjs/issues/3735#issuecomment-913921482)

**Comment by leejroberts** [comment]
:+1:

File: src/style_manager/view/PropertyIntegerView.js [hunk 3]
```diff
@@ -5,7 +5,21 @@ const $ = Backbone.$;
...
-    return '';
+    const pfx = this.pfx;
+    const ppfx = this.ppfx;
+    const icon = this.model.get('icon');
+    const textLabel = this.model.get('textLabel');
+    const iconEl = icon
+      ? `<i class="${icon}"></i>`
+      : textLabel
+      ? `<div>${textLabel}</div>`
+      : '';
```
Comment (leejroberts) [inline src/style_manager/view/PropertyIntegerView.js:16]: feel free to buck his coding style. likely eventually we will rewrite the whole thing lol

**Review by leejroberts — APPROVED** [review]
You're doing the lords work here. Vaya con dios!

**Pull request merged**